.github/ISSUE_TEMPLATE/custom_widget_issue.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Custom Widget
+about: Issues or questions related to developing a custom widget
+labels: type:custom-widget
+---
+
+<!-- 
+If this is a:
+
+Bug Report: Please add the relevant pieces of information from the bug report template to this issue  (https://github.com/jupyterlab/jupyterlab/issues/new?template=bug_report.md)
+
+Question: Great! Thanks for asking someone else will be able to find the answer more easily in the future. Please try to provide as much detail as you can about what you would like to accomplish and what you have tried.
```

Comment:
```suggestion
Question: Great! Thanks for asking here, since someone else will be able to find the answer more easily in the future. Please try to provide as much detail as you can about what you would like to accomplish and what you have tried.
```